### ISSUE TITLE
epsilon/0, pi/0, and e/0 wrongly reported as built-in predicates

### ISSUE THREAD

**Issue by pmoura** [infradig/trealla#264]
For example:

```text
$ tpl
Trealla Prolog (c) Infradig 2020-2021, v1.13.25-1-gf3301
?- predicate_property(epsilon, P).
P = built_in 
; P = static 
; P = private 
; P = native_code 
; ?- epsilon.
uncaught exception: error(existence_error(procedure,epsilon/0),epsilon/0)
?- 
```

**Comment by pmoura** [comment]
The built-in arithmetic functions seem also to be wrongly reported as built-in predicates. For example:

```text
?- predicate_property(exp(_), P).
P = built_in 
; P = static 
; P = private 
; P = native_code 
; ?- exp(E).
uncaught exception: error(existence_error(procedure,exp/1),exp/1)
?- 
```

**Comment by pmoura** [comment]
Re-tested with v1.13.26. The fix is partial, however, as properties are still returned for arithmetic constants and arithmetic functions, which are not predicates (my bad as I should have been more clear in my bug report).

**Comment by pmoura** [comment]
Fix confirmed using v1.13.31. Thanks.
